src/ui/screens/ProposalScreen.tsx

```diff
@@ -19,7 +19,7 @@ import globalStyles, {standardPadding} from '@ui/styles';
 import {Button, Caption, Headline, Icon, List, Modal, Text} from '@ui/library';
 import {Layout} from '@ui/components/Layout';
 
-export function DemocracyProposalScreen({route}: {route: RouteProp<DashboardStackParamList, typeof referendumScreen>}) {
```

Comment:
Maybe we should keep this, there are also Treasury proposals. We currently don't have any screen for that but it makes it more clear